packages/api/endpoints/workflows.js

```diff
@@ -1,22 +1,39 @@
 'use strict';
 
-const _get = require('lodash.get');
-const { S3 } = require('@cumulus/ingest/aws');
+const aws = require('@cumulus/common/aws');
 const handle = require('../lib/response').handle;
 
+/**
+ * Get S3 object
+ *
+ * @returns {undefined} undefined
+ */
+async function getWorkflowList() {
+  const workflowsListKey = `${process.env.stackName}/workflows/list.json`;
+  try {
+    const { Body } = await aws.getS3Object(process.env.bucket, workflowsListKey);
+    return Body;
+  }
+  catch (err) {
+    return err;
+  }
+}
+
 /**
  * List all providers.
  *
  * @param {Object} event - aws lambda event object.
  * @param {callback} cb - aws lambda callback function
  * @returns {undefined} undefined
  */
-function list(event, cb) {
-  const key = `${process.env.stackName}/workflows/list.json`;
-  S3.get(process.env.bucket, key).then((file) => {
-    const workflows = JSON.parse(file.Body.toString());
-    return cb(null, workflows);
-  }).catch((e) => cb(e));
+async function list(event, cb) {
```

Comment:
Since the `event` argument isn't being used, it should be removed from the `list()` function.